servers/quarkus-server/src/main/resources/application.properties

```diff
@@ -132,15 +127,14 @@ quarkus.container-image.name=nessie
 %test.nessie.backends.type=INMEMORY
 %test.nessie.version.store.jgit.type=INMEMORY
 
-mp.openapi.extensions.smallrye.info.title=Example API
-%dev.mp.openapi.extensions.smallrye.info.title=Example API (development)
-%test.mp.openapi.extensions.smallrye.info.title=Example API (test)
-mp.openapi.extensions.smallrye.info.version=1.0.1
-mp.openapi.extensions.smallrye.info.description=Just an example service
-mp.openapi.extensions.smallrye.info.termsOfService=Your terms here
-mp.openapi.extensions.smallrye.info.contact.email=techsupport@example.com
-mp.openapi.extensions.smallrye.info.contact.name=Example API Support
-mp.openapi.extensions.smallrye.info.contact.url=http://exampleurl.com/contact
+mp.openapi.extensions.smallrye.info.title=Project Nessie API
```

Comment:
I was thinking of adding those in model using the annotations. Thoughts?

Reply:
yes, that makes a lot more sense imho

Reply:
I've removed these lines. Want to add your change to this patch or separately?